SliderButton/tests/SliderButton-specs.js

```diff
@@ -81,4 +81,34 @@ describe('SliderButton Specs', () => {
 		const actual = handleChange.mock.calls.length;
 		expect(actual).toBe(expected);
 	});
+
+	test('should change value on arrow right key press', () => {
+		const handleChange = jest.fn();
+		const sliderButton = mount(
+			<SliderButton onChange={handleChange}>
+				{children}
+			</SliderButton>
+		);
+
+		sliderButton.find('SliderButton').simulate('keyDown', {which: 39, keyCode: 39, code: 39});
```

Comment:
I think only "keyCode" is sufficient. Strangely, only Input used it like this. I don't know if there is a reason

Reply:
👍